internal/preload/instance_type.go

```diff
@@ -79,7 +79,7 @@ func (p *instanceType) ValidateRegion(region string) bool {
 		panic(fmt.Errorf("unable to read availability dir: %w", err))
 	}
 
-	fullName := region + ".yaml"
+	fullName := region + "_1.yaml"
```

Comment:
:warning: change for the future, we now validate region, not location.

In the past, we've validated for location, but then actually ignored the param down the line and forced `eastus`, but we actually need regions, not locations in all the calls (even tho azure calls them Location :facepalm:)